packages/esm-patient-conditions-app/src/conditions/conditions-form.component.tsx

```diff
@@ -51,6 +51,11 @@ const ConditionsForm: React.FC<ConditionFormProps> = ({ closeWorkspace, conditio
   });
 
   const onSubmit = (data) => {
+    if (!data.search) {
+      setIsSubmittingForm(false);
+      setErrorCreating({ message: 'Please choose a condition' });
```

Comment:
Why aren't we adding the validation via RHF, since it's being used in the form.

Also, why do we have a separate state, when `setError` from `useForm` is present 🤔

Reply:
Noted that and I am working on it